src/Tribe/Views/V2/Admin/Bar.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Handles Admin Bar management for Views V2.
+ *
+ * @since   5.0.0
+ *
+ * @package Tribe\Events\Views\V2\Admin
+ */
+
+namespace Tribe\Events\Views\V2\Admin;
+
+use Tribe__Utils__Array as Arr;
+use WP_Admin_Bar;
+
+/**
+ * Class Bar
+ *
+ * @since   5.0.0
+ *
+ * @package Tribe\Events\Views\V2\Admin
+ */
+class Bar {
+	CONST SUSPEND_CACHE_KEY = 'tribe_events_suspend_view_html_cache';
+	CONST SUSPEND_NONCE_KEY = 'tribe_events_views_v2_suspend_view_html_cache';
+
+	/**
+	 * Check if the current user has view html cache suspended.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return  boolean  If the current user has suspended the cache for views.
+	 */
+	public function is_view_html_cache_suspended() {
+		// Check anything if you have WP_DEBUG turned on.
+		if ( ! defined( 'WP_DEBUG' ) ) {
+			return false;
+		}
+
+		// You can only suspend when you can `manage_options`
+		if ( ! current_user_can( 'manage_options' ) ) {
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 21935f25ebcee516a7d1168c3dc27b24</sub>